facade-app/src/test/CommandInjection.test.tsx

```diff
@@ -4,13 +4,13 @@ import { render, screen, waitFor } from "@testing-library/react";
 import { Content } from "../Components/Content";
 import testFixture from "./fixtures";
 
-jest.mock('../Utilities/Utils', () => {
+jest.mock("../Utilities/Utils", () => {
```

Comment:
One question, @Dripcoding do we need to do this for each type of vulnerability? There are 12-15 types and may be more in future.

Reply:
yes we'll have to do it for each test-suite. I can work on centralizing the unit test mocks in my next PR.

Reply:
sure, we can move this one to central place. I have question that we are adding commandInjection.test.tsx and similarly we are adding for unrestrictedFileUpload so do we need to add for all? All this data gets generated from JSON so i think having 1-2 should be sufficient. thoughts?

Reply:
That is a good point. Usually one either tries to write tests covering each permutation of a component or write tests that sufficiently cover the component logic given some input. I usually like to do the former when I can.

As a middle ground I can try to create a single test suite using the different vulnerability JSON payloads as parameters.

Reply:
Makes sense. please go ahead with what ever makes you more comfortable.